lib/griddler/email.rb

```diff
@@ -79,6 +76,19 @@ def clean_html(html)
       cleaned_html
     end
 
+    def deep_clean_invalid_utf8_bytes(o)
+      case o
+      when Hash
+        o.inject({}) { |h, (k, v)| h[k] = deep_clean_invalid_utf8_bytes(v); h }
```

Comment:
Prefer `reduce` over `inject`.<br>Do not use semicolons to terminate expressions.